### PR TITLE
Change dependencies documentation opencv2-->opencv

### PR DESCRIPTION
**Pull request by aaronmarkham**
## Description ##
OpenCV version changed from `opencv@2` to `opencv`.
I also deleted a duplicated section of content.

### CODE REVIEW

**Comment by vandanavk** [comment]
@mxnet-label-bot[pr-awaiting-review]

**Comment by vandanavk** [comment]
@aaronmarkham Could you check the CI build failure?

File: docs/tutorials/scala/mxnet_scala_on_intellij.md [hunk 1]
```diff
@@ -340,26 +340,13 @@ If you get an error, check the dependencies at the beginning of this tutorial. F
 
 ```
 ...
-Library not loaded: /usr/local/opt/opencv@2/lib/libopencv_calib3d.2.4.dylib
+Library not loaded: /usr/local/opt/opencv/lib/libopencv_calib3d.2.4.dylib
```
Comment (lanking520) [inline docs/tutorials/scala/mxnet_scala_on_intellij.md]: Maybe not 2.4 since this is Opencv3... change 2.4 into x.x maybe better

**Comment by vandanavk** [comment]
@sandeep-krishnamurthy This PR has been approved and is ready for merge

**Comment by lanking520** [comment]
> @sandeep-krishnamurthy This PR has been approved and is ready for merge

@vandanavk Just a note: MXNet repo usually requires at least two committers/contributors approval in order to get a PR merged.

**Comment by sandeep-krishnamurthy** [comment]
@lanking520 - I reviewed the changes before merging. Missed to add approval. Thanks for the nudge will take care going forward.

**Pull request merged**